server/src/main/java/org/tango/server/cache/TangoCacheManager.java

```diff
@@ -400,6 +401,8 @@ public synchronized SelfPopulatingCache getAttributeCache(final AttributeImpl at
             AttributeCache attrCache = attributeCacheMap.get(attr);
             if (attrCache == null) {
                 attrCache = extTrigAttributeCacheMap.get(attr);
+                if (attrCache == null)
```

Comment:
Too many nested ifs. I will move it into a separate method.

Reply:
True. But my goal was to fix the problem quickly and not to change the code itself.